test/files/jvm/patmat_opt_no_nullcheck.check

```diff
@@ -1 +1,26 @@
-bytecode identical
```

Comment:
Mhh, `ASMConverters.equivalentBytecode` didn't work (without digging any deeper).

I'll move it to pending for now.